### ISSUE TITLE
Timeslider Realtime Playback for Video-Sync

### ISSUE THREAD

**Issue by BenjaminBeichler** [ether/etherpad-lite#5012]
I'm using BigBlueButton for teaching, and this includes a "shared notes" feature, which is an embedded Etherpad. 

I offer recordings for my lectures and since I also use the etherpad to show the path to a solution, it would be nice to allow the realtime playback of the etherpad. Currently the delay between changes in the timeslider playback seems to be more or less constant, which is of course hard to follow with a parallel recorded audio session.

It is easily possible to export the shared notes in the .etherpad format, which seems to include real timestamps of the changes.

I tried to import the .etherpad file, which seems to work fine, but the timeslider is to fast in the playback. Is it possible via some URL-Parameters to change the behavior of the timeslider? Or would it be a suggest for a new feature to allow this realtime-playback?

**Comment by webzwo0i** [comment]
afaik there's no such setting. In https://github.com/ether/etherpad-lite/blob/develop/src/static/js/broadcast_slider.js#L181 the delay is hard-coded to 100ms.

I suppose it's a new feature, ie something like "playback in original edit time".

**Comment by BenjaminBeichler** [comment]
I cannot assign the feature-request label, but is this issue enough as a feature request, or should I open an extra issue?

**Comment by JohnMcLear** [comment]
Sounds like a sensible feature, "realtime playback speed" would be a nice name for it.  I guess it would be a checkbox under settings and I guess just stepping at ``rev2:timestamp[int]-rev1:timestamp[int]``ms instead of the fixed 100ms value would do the job.

Another sensible option might be to have a dropdown/select in settings which provides options for playback speed IE realtime, 100ms, 200ms...

I don't know if this could be a plugin so would need to be core.

@BenjaminBeichler the best way to get us to add stuff like this is to get people to vote/comment on this issue.

**Comment by dcht00** [comment]
Big +1 from me! :)

What seems key to me is to be able, via URL parameters:
* enable realtime
* to jump to a specific time (or I suppose "first revision after") 
* autostart

Maybe not the best place to mention this, but for this feature to sync well, another timeslider behaviour might need to be addressed. On bigger pads, I noticed pressing "Play" starts to count-off the revisions immediately - but the content stays frozen (at last revision). It can take 10 seconds or more to start following the revisions.

**Comment by JohnMcLear** [comment]
I like the idea of realtime playback..   I don't know if it needs to be 

a) server wide
b) user setting <--- I think this
c) plugin or feature...

It should be fun to work on anyway :)

**Comment by JohnMcLear** [comment]
I really like this feature, it's a shame it took 5 years to get around to building it as it was only 10 mins or so work to implement!

https://github.com/ether/etherpad/pull/7541/
